README.md

```diff
@@ -96,6 +103,10 @@ Since release 0.7:
 * Heap abstract domain with intervals or zones: --heap-[intervals|zones]
 * Heap abstract domain with intervals or zones and loop paths: --heap-[intervals|zones] --sympath
 
+Since release 0.10:
+
+* Array abstract domain: --arrays
+
```

Comment:
There are more publications:
Viktor Malík, Peter Schrammel, Tomás Vojnar:
2LS: Heap Analysis and Memory Safety - (Competition Contribution). TACAS (2) 2020: 368-372

Reply:
True. I'll fix that directly in master if you don't mind.